dev/pages/Button.vue

```diff
@@ -30,7 +30,7 @@ const modifiers = reactive(buildCheckboxState({ controls: modifierControls }))
       <w-button
         :primary="active('Primary')"
         :secondary="active('Secondary')"
-        :negative="modifiers.Negative"
+        :negative="active('Negative')"
```

Comment:
I support aligning this across framework implementations 🙌  I think we should then have a clear note in the changelog that `negative` attribute can no longer be used as a modifier that e.g. overrides primary button colors. It could be it was used this way. From now on users will need to replace any other variant that was set on w-button with that negative variant if they want to use it.

Reply:
As it is stated in the docs 👇, `negative` is a modifier for a `primary` button. Thank you for pointing that. I just fixed how it is on other frameworks but maybe I was too quick:
<img width="1100" alt="image" src="https://github.com/warp-ds/vue/assets/37986637/7e4a53fb-81f3-428e-9d89-1920b7635068">

DO you think I should align with the docs and original implementation OR with other frameworks, where negative is a button type. I don't think we need a breaking change for now 😓

Reply:
I think `negative` should be a button type/variant, so a button can be either `primary` or `negative` across all our implementations. There's no examples of Vue Button being migrated from Fabric atm, so we should be good only mentioning this change in the Changelog. I think we might need to avoid adding `breaking change` in the commit message to not trigger a major version update by semantic release.